test/Microsoft.AspNet.Mvc.IntegrationTests/FormCollectionModelBindingIntegrationTest.cs

```diff
@@ -147,7 +147,6 @@ private class Address
             // No data is passed.
             var operationContext = ModelBindingTestHelper.GetOperationBindingContext(request =>
             {
-                request.ContentType = "multipart/form-data";
```

Comment:
put another way, multipart isn't wrong as long as the form content is in the main part